wdtk-dumpfiles/src/main/java/org/wikidata/wdtk/dumpfiles/WikibaseRevisionProcessor.java

```diff
@@ -154,6 +164,10 @@ public void processPropertyRevision(MwRevision mwRevision) {
 
 	}
 
+	private boolean isWikibaseRedirection(MwRevision mwRevision) {
+		return redirectionsPattern.matcher(mwRevision.getText()).matches();
```

Comment:
Given that one of the main goals of this code is to be as fast as possible, I believe this should not use a regular expression but a `startsWith` or something similar.